src/main/java/com/it4045/pokemonviewer/enterprise/dto/Pokemon.java

```diff
@@ -25,8 +26,8 @@
     private List<Evolution> pokemonPrevEvolution;
 
     public Pokemon(){ }
-
-    public Pokemon(String pokemonName, String image, List<String> weakness, List<String> type, String spawnTime) {
+    
+    public Pokemon(String pokemonName, String image, List<String> weakness, List<PokemonType> type, String spawnTime) {
```

Comment:
Constructor looks good. I changed List type to use the new PokemonType enum. I did some research and believe pokemon can be more than one type if this is wrong you can simply make pokemonType a variable type of PokemonType instead of a List<PokemonType>